create.sh

```diff
@@ -13,13 +13,12 @@ if [[ -z "$RUNNER_URL" ]]; then
 fi
 
 # Deploy infrastructure
-PROJECT_NAME=${PROJECT_NAME} scripts/deploy_infrastructure.sh
+TF_VAR_project_id=${PROJECT_ID} scripts/deploy_infrastructure.sh
```

Comment:
I don't think all these vars should be `TF_VAR_project_id`, they should just be `PROJECT_ID=${PROJECT_ID}` since it is not exported and it is just being passed around hence using `TF_VAR` has no use here?